ltsp/server/isc-dhcp-server/maas-dhcpd-snippet.conf

```diff
@@ -0,0 +1,46 @@
+# This file is part of LTSP, https://ltsp.github.io
+# Copyright 2019 the LTSP team, see AUTHORS
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# MAAS isc-dhcp-server snippet for LTSP
+# in order to boot LTSP in a subnet add a global DHCP snippet and one to the
+# subnet, which you desire to enbale LTSP on.
```

Comment:
Do we really want to ship a lot of isc-dhcp snippets based on what framework people are using?
If we can ship only one, it would make sense.
If more are needed, then I think the wiki is a more appropriate place to host them...

Reply:
@alkisg, I fully agree, we should not feel compelled to provide configs for any possible scenario.

However, I propose to include the full isc-dhcp-server example in the package and reference it in the man page for people to find more easily.

My version with the "cut marks" for MAAS snippets probably is better suited for the Wiki.